### PR TITLE
Added PHP 8.0, 7.4, and 7.3 to HelioHost

### PR DESCRIPTION
**Pull request by Kryd0s**
Added PHP 8.0, PHP 7.4, and PHP 7.3 to HelioHost, manually updated the older versions, and changed http to https.

### CODE REVIEW

File: _data/hosts.yml [hunk 1]
```diff
@@ -2958,38 +2958,53 @@
...
         55:
-            phpinfo: 'http://krydos.heliohost.org/55/phpinfo.php'
+            phpinfo: 'https://krydos.heliohost.org/55/phpinfo.php'
             patch: 38
             version: 5.5.38
             semver: 5.5.38
         56:
-            phpinfo: 'http://krydos.heliohost.org/56/phpinfo.php'
+            phpinfo: 'https://krydos.heliohost.org/56/phpinfo.php'
             patch: 30
```
Comment (colinodell) [inline _data/hosts.yml:2975]:
```suggestion
            patch: 40
```

File: _data/hosts.yml [hunk 2]
```diff
@@ -2958,38 +2958,53 @@
...
-            version: 5.6.30
-            semver: 5.6.30
+            version: 5.6.40
+            semver: 5.6.40
         70:
-            phpinfo: 'http://krydos.heliohost.org/70/phpinfo.php'
+            phpinfo: 'https://krydos.heliohost.org/70/phpinfo.php'
             patch: 18
```
Comment (colinodell) [inline _data/hosts.yml:2980]:
```suggestion
            patch: 33
```

File: _data/hosts.yml [hunk 3]
```diff
@@ -2958,38 +2958,53 @@
...
-            version: 7.0.18
-            semver: 7.0.18
+            version: 7.0.33
+            semver: 7.0.33
         71:
-            phpinfo: 'http://krydos.heliohost.org/71/phpinfo.php'
+            phpinfo: 'https://krydos.heliohost.org/71/phpinfo.php'
             patch: 4
```
Comment (colinodell) [inline _data/hosts.yml:2985]:
```suggestion
            patch: 33
```

File: _data/hosts.yml [hunk 4]
```diff
@@ -2958,38 +2958,53 @@
...
-            version: 7.1.4
-            semver: 7.1.4
+            version: 7.1.33
+            semver: 7.1.33
         72:
-            phpinfo: 'http://krydos.heliohost.org/72/phpinfo.php'
+            phpinfo: 'https://krydos.heliohost.org/72/phpinfo.php'
             patch: 0
```
Comment (colinodell) [inline _data/hosts.yml:2990]:
```suggestion
            patch: 34
```

File: _data/hosts.yml [hunk 5]
```diff
@@ -2958,38 +2958,53 @@
...
-            version: 7.2.0
-            semver: 7.2.0
+            version: 7.2.34
+            semver: 7.2.34
+        73:
+            phpinfo: 'https://krydos.heliohost.org/73/phpinfo.php'
+            patch: 0
```
Comment (colinodell) [inline _data/hosts.yml:2995]:
```suggestion
            patch: 28
```

File: _data/hosts.yml [hunk 6]
```diff
@@ -2958,38 +2958,53 @@
...
+            version: 7.3.28
+            semver: 7.3.28
+        74:
+            phpinfo: 'https://krydos.heliohost.org/74/phpinfo.php'
+            patch: 0
```
Comment (colinodell) [inline _data/hosts.yml:3000]:
```suggestion
            patch: 19
```

File: _data/hosts.yml [hunk 7]
```diff
@@ -2958,38 +2958,53 @@
...
+            version: 7.4.19
+            semver: 7.4.19
+        80:
+            phpinfo: 'https://krydos.heliohost.org/80/phpinfo.php'
+            patch: 0
```
Comment (colinodell) [inline _data/hosts.yml:3005]:
```suggestion
            patch: 6
```

**Review by colinodell — CHANGES_REQUESTED** [review]
Would you mind setting the `patch` numbers as well? Thanks!

**Review by colinodell — APPROVED** [review]
TYVM!

**Comment by Kryd0s** [comment]
Thanks for the merge.

**Pull request merged**